rs/execution_environment/src/execution/call_or_task/tests.rs

```diff
@@ -115,6 +115,12 @@ fn wat_writing_to_each_stable_memory_page_query(memory_amount: u64) -> String {
     )
 }
 
+// Helper function to allow testing both update and query methods with the same test.
+fn with_update_and_replicated_query<F: FnOnce(String) + Clone>(test: F) {
+    test.clone()("update".to_string());
+    test("query".to_string());
+}
```

Comment:
```suggestion
fn with_update_and_replicated_query<F: Fn(&str)>(test: F) {
    test("update");
    test("query");
}
```
I tried it for a single test and that one worked. But because of the `String` -> `&str` change you'll have to remove a `&` in some tests.

Reply:
I can live with removing `&`, thanks for the suggestion.

Reply:
Wait, you misunderstood the problem here. The clone is on the function, not the string. The compiler somehow doesn't allow me to reuse the test function as it considers it moved once I call it once.